features/fixtures/mazerunner/src/main/java/com/bugsnag/android/mazerunner/scenarios/HandledKotlinSmokeScenario.kt

```diff
@@ -0,0 +1,41 @@
+package com.bugsnag.android.mazerunner.scenarios
+
+import android.content.Context
+import com.bugsnag.android.*
+import java.util.*
+
+/**
+ * Sends a handled exception to Bugsnag, which does not include session data.
+ */
+internal class HandledKotlinSmokeScenario(config: Configuration,
+                                        context: Context) : Scenario(config, context) {
+    init {
+        config.autoTrackSessions = false
```

Comment:
Kotlin has a pretty cool `with` statement where anything in the scope wouldn't have to call `config.`: https://kotlinlang.org/docs/reference/scope-functions.html#with

Reply:
I'll add a ticket to refactor the Android test fixture, I'm sure there's a bunch of things in there which could be changed